client/lib/Screens/login/login_screen.dart

```diff
@@ -0,0 +1,145 @@
+import 'package:flutter/material.dart';
+
+class LoginScreen extends StatelessWidget {
+  const LoginScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: ListView(
+        padding: EdgeInsets.symmetric(
+            horizontal: MediaQuery.of(context).size.width / 8),
+        children: const [
+          Menu(),
+          Body(),
+        ],
+      ),
+    );
+  }
+}
+
+class Menu extends StatelessWidget {
+  const Menu({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 30),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.start,
+        children: [
+          Image.asset(
+            'icons/release_icon_transparent.png',
+            width: 41,
+            height: 65,
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class Body extends StatelessWidget {
+  const Body({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    TextTheme texttheme = Theme.of(context).textTheme;
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        SizedBox(
+          width: 380,
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Text(
+                '   Sign In to the\nCommunity Portal',
+                style: texttheme.headline1,
+              ),
+              Text(
+                "If you don't have an account",
+                style: texttheme.subtitle2,
+              ),
+              Row(
+                children: [
+                  RichText(
+                    text: TextSpan(
+                      text: 'You can',
+                      style: texttheme.subtitle2,
+                    ),
+                  ),
+                  Text(
+                    ' Register here!',
+                    style: TextStyle(
+                        color: Theme.of(context).primaryColor,
+                        fontWeight: FontWeight.bold),
+                  ),
+                ],
+              ),
+              Image.asset(
+                'icons/release_icon_with_name_transparent.png',
```

Comment:
Please have an external class with all assets.

Reply:
you mean creating a widget for images?

Reply:
No, a class listing all barcoded assets paths

Reply:
sorry ,but that mean?

Reply:
`class Resources {
  const Resources._();
  static final String icon = 'icons/logo.png';
}`
Is that you were taking about.

Reply:
Yes exactly